docs/content/steps/adding-editing-steps.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: Adding or editing workflow steps
+title: Adding or editing steps from apps
 lang: en
 slug: /concepts/adding-editing-steps
 ---
 
 :::danger
 
-Workflow Steps from Apps are a deprecated feature.
+Steps from Apps are a deprecated feature.
 
-Workflow Steps from Apps are different than, and not interchangable with, Slack automation workflows. We encourage those who are currently publishing Workflow Steps from Apps to consider the new [Slack automation features](https://api.slack.com/automation), such as [custom functions for Bolt](/concepts/creating-custom-functions).
+Steps from Apps are different than, and not interchangable with, Slack automation workflows. We encourage those who are currently publishing steps from apps to consider the new [Slack automation features](https://api.slack.com/automation), such as [custom steps for Bolt](/concepts/creating-custom-functions).
```

Comment:
```suggestion
Steps from Apps are different than, and not interchangable with, Slack automation workflows. We encourage those who are currently publishing steps from apps to consider the new [Slack automation features](https://api.slack.com/automation), such as [custom steps for Bolt](/concepts/custom-steps).
```

This might be optimistic for this PR, but I think it should be included for the changes in #2198!

Reply:
Good catch `/concepts/custom-step` is not out yet I will link to https://api.slack.com/automation/functions/custom-bolt instead